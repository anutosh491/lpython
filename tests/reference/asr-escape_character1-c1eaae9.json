{
    "basename": "asr-escape_character1-c1eaae9",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/escape_character1.py",
    "infile_hash": "1eb54c099258db254defa83efbf30345041207d7a90b9a56a679d20c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-escape_character1-c1eaae9.stdout",
    "stdout_hash": "1633c41c2718858b9d2e45ae51d6d05c01253b2650f7ffe39cf388f6",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}