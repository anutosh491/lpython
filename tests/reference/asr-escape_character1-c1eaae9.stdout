(TranslationUnit (SymbolTable 1 {_lpython_main_program: (Function (SymbolTable 5 {}) _lpython_main_program (FunctionType [] () Source Implementation () .false. .false. .false. .false. .false. [] [] .false.) [main0] [] [(SubroutineCall 1 main0 () [] ())] () Public .false. .false.), main0: (Function (SymbolTable 3 {}) main0 (FunctionType [] () Source Implementation () .false. .false. .false. .false. .false. [] [] .false.) [testEscapeCharacters] [] [(SubroutineCall 1 testEscapeCharacters () [] ())] () Public .false. .false.), main_program: (Program (SymbolTable 4 {}) main_program [] [(SubroutineCall 1 _lpython_main_program () [] ())]), testEscapeCharacters: (Function (SymbolTable 2 {s1: (Variable 2 s1 [] Local () () Default (Character 1 -2 () []) Source Public Required .false.), s2: (Variable 2 s2 [] Local () () Default (Character 1 -2 () []) Source Public Required .false.), s3: (Variable 2 s3 [] Local () () Default (Character 1 -2 () []) Source Public Required .false.)}) testEscapeCharacters (FunctionType [] () Source Implementation () .false. .false. .false. .false. .false. [] [] .false.) [] [] [(= (Var 2 s1) (StringConstant "Hello	World" (Character 1 11 () [])) ()) (= (Var 2 s2) (StringConstant "Hello    World" (Character 1 15 () [])) ()) (= (Var 2 s3) (StringConstant "HelloWorld" (Character 1 11 () [])) ()) (Print () [(Var 2 s1)] () ()) (Print () [(Var 2 s2)] () ()) (Print () [(Var 2 s3)] () ())] () Public .false. .false.)}) [])
